user/cinnamon-settings-daemon/template.py

```diff
@@ -0,0 +1,50 @@
+pkgname = "cinnamon-settings-daemon"
+pkgver = "6.2.0"
+pkgrel = 0
+build_style = "meson"
```

Comment:
```suggestion
build_style = "meson"
configure_args = ["-Ddefault_library=shared"]
```
and drop -static subpkg